### PR TITLE
Fix the Downtime Quick Link #97

### PR DESCRIPTION
**Pull request by RONNCC**

### CODE REVIEW

**Comment by pkoenig10** [comment]
Sorry I wasn't clear.  There are other non-building categories besides "Non-Building" (SCC and Staff).  To make sure this behavior is correct even if we add more categories, we should choose to display Fraternity, Sorority, Independent, Blitz, and Concessions, rather than no choosing not to display orgs that are non-building.

**Comment by RONNCC** [comment]
Ah thanks! You hit the nail on the head :)

**Comment by RONNCC** [comment]
that should be fixed now - @purelogiq can you look at this ?

**Comment by purelogiq** [comment]
looking

File: app/models/organization.rb [hunk 1]
```diff
@@ -38,6 +38,8 @@ class Organization < ActiveRecord::Base
 
   default_scope { order('name asc') }
 
+  scope :only_categories, lambda {|category_name_array| joins(:organization_category).where(organization_categories: {name: category_name_array}) }
+  scope :by_not_category, lambda {|category_name| joins(:organization_category).where.not(organization_categories: {name: category_name}) }
```
Comment (purelogiq) [inline app/models/organization.rb]: Remove the :by_not_category scope. There is no need for unused code and it's a simple fix.

**Comment by purelogiq** [comment]
Also ronnie please modify line 8 in `app/controllers/organization_timeline_entries_controller.rb` 

Currently it reads:
`@entries = OrganizationTimelineEntry.entry_type.downtime`  

But it should read
`@entries = OrganizationTimelineEntry.downtime`

This was causing the index page for org downtimes to crash. You can get to this page by going to the downtime page and clicking one of the org's names.

This small bug is highly related and I think it is appropriate to group it with this fix.

**Comment by RONNCC** [comment]
fixed :)

**Comment by purelogiq** [comment]
Looks good, will do 1 final check before merging.

**Pull request merged**